scripts/register

```diff
@@ -19,22 +19,25 @@ Options:
   -s amount     an optional amount of stake to set up (by default 20)
   -a account    an optional account to use for funding and staking (it requires the mnemonic parameter)
   -m mnemonic   an optional mnemonic to use for unlocking the account parameter (it requires the account parameter)
+  -c path       an optional path to a file containing an alternative server configuration
   -h            show this help message"
 
 LONG_HELP="Usage: $PROG [options]
 Options:
-  -f | --funds      amount    an optional amount of funds to set up (by default 10)
-  -s | --output     amount    an optional amount of stake to set up (by default 20)
-  -a | --account    account   an optional account to use for funding and staking (it requires the mnemonic parameter)
-  -m | --mnemonic   mnemonic  an optional mnemonic to use for unlocking the account parameter (it requires the account parameter)
-  -h | --help                 show this help message"
+  -f | --funds        amount    an optional amount of funds to set up (by default 10)
+  -s | --output       amount    an optional amount of stake to set up (by default 20)
+  -a | --account      account   an optional account to use for funding and staking (it requires the mnemonic parameter)
+  -m | --mnemonic     mnemonic  an optional mnemonic to use for unlocking the account parameter (it requires the account parameter)
+  -c | --config_file  path      an optional path to a file containing an alternative server configuration
```

Comment:
I would specify here the default value, something like: `(by default server-config.json)`

Reply:
Yes, good one. Done.